xpdan/tools.py

```diff
@@ -240,3 +240,27 @@ def decompress_mask(data, indices, indptr, shape):
     cmask = csr_matrix(
         tuple([np.asarray(a) for a in [data, indices, indptr]]), shape=shape)
     return ~cmask.toarray().astype(bool)
+
+
+def correct_polarization(geo, img, polarization_factor):
+    return img/geo.polarization(img.shape, polarization_factor)
+
+
+def subtract(arr1, arr2):
```

Comment:
No need for this; it would be better to use the built-in. https://docs.python.org/2/library/operator.html#operator.sub